pysolr.py

```diff
@@ -300,6 +303,16 @@ def __iter__(self):
             result = result._next_page_query and result._next_page_query()
 
 
+def get_nested(obj, keys, default=None):
```

Comment:
It looks like this is only used by a debug logging call, and it could be replaced by a standard dictionary get-with-default call there.